### ISSUE TITLE
background should not scroll for mat-dialog if opened from mat-menu

### ISSUE THREAD

**Issue by dilyanapavlova** [angular/material2#12091]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
The dialog background shouldn't scroll.

#### What is the current behavior?
The background does scroll, when mat-dialog is opened from mat-menu.

#### What are the steps to reproduce?
https://stackblitz.com/edit/angular-material2-issue-4ljghh

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
@latest

#### Is there anything else we should know?
We use scrollstrategy "block" for mat-menu to prevent scrolling of the background when the menu is open (bug: mat-menu scroll)

**Comment by jelbourn** [comment]
@crisbeto I took a look at the reproduction and something does seem off.

**Comment by silvioboehme** [comment]
Hi! We have basically the same problem:
Regarding to an other bug we changed to scroll strategy for the menu to BLOCK.

Now if you open a dialog from the menu, the scroll strategy for the dialog (which is block!) is overridden, and the page is scrolling behind the dialog. If we leave the scroll strategy for the menu to default, the dialog is working correctly.

Thank you!

**Comment by rudolfolah** [comment]
Agreed with this, I *just* noticed it when working on a login/signup component and realizing that in tablet view the background can still scroll. A 100% width/height login on mobile is fine, a small 40%/40% is fine on desktop, but on tablet...it's not a good result.

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
